src/ferui-design/scss/components/widget/_widget.scss

```diff
@@ -0,0 +1,89 @@
+@import '../../variables';
+@import '../../mixins';
+
+@include exports('components.widget') {
+  .fui-widget {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    background-color: $white;
+    border: 1px solid $gray-300;
+    border-radius: 3px;
```

Comment:
use $default-radius-value or $border-radius variables for border radius value pls.